driver-adapters-wasm/d1-cfpages-nuxt/wrangler.toml

```
@@ -0,0 +1,8 @@
+#:schema node_modules/wrangler/config-schema.json
+name = "prisma-driver-adapters-nuxt"
+compatibility_date = "2024-04-05"
+
```

You can add this to make sure it's the right account id 
```suggestion
account_id = "3dc90bc6bf50e78795c6f70865397810"
```

---

It's the ID for the Ecosystem test account